### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SvetlanaMikheeva**

### CODE REVIEW

**Review by dmitry-eliseev-devexpress — APPROVED** [review]
**Structure and Semantics**

This is a simple step-by-step instruction without any structural or semantic issues. The topic serves its purpose well. Well done.

**Language**

In general, your language is good, however, I made a few stylistic changes as well as couple definiteness edits.

How can you identify if you need a definite article for an object defined by that? Can you name those fields in this case what they are? Yes, you can. What fields? All fields that failed validation. In this case, "all" is a fully-functional equivalent of the definite article.

Regarding the word "corresponding", it's too heavy to omit a definite article in general because this word sets unambiguous connections between two entities. What error messages? The corresponding error messages. That is the messages that correspond to the fields that failed validation.

As you can see, everything is not that difficult in this case.

**Bottom Line**

This is a good topic. I can approve it right away.
